web/src/app/admin/assistants/AssistantEditor.tsx

```diff
@@ -342,7 +339,12 @@ export function AssistantEditor({
 
           if (imageGenerationToolEnabled) {
             if (
-              !checkLLMSupportsImageInput(
+              !checkLLMSupportsImageOutput(
```

Comment:
Possibly a similar issue with this function too ... is there some sort of central file where we define all the different models and capabilities?